doc/releases/changelog-0.11.0.md

```diff
@@ -177,21 +177,36 @@
       return circuit(x)
   ```
 
-* Changes to reduce compile time:
+  ```pycon
+  >>> func(0.1)
+  Array(0.99500417, dtype=float64)  
+  ```
+
+<h3>Improvements 🛠</h3>
 
-  - Turn off MLIR's verifier.
+* Pattern rewriting in the `quantum-to-ion` lowering pass has been changed to use MLIR's dialect conversion
+  infrastructure.
+  [(#1442)](https://github.com/PennyLaneAI/catalyst/pull/1442)
+
+* The `merge-rotations` peephole optimization pass has been extended to also merge compatible rotation 
+  gates (either both controlled, or both uncontrolled) where rotation angles are any combination of 
+  static constants or dynamic values.
+  [(#1489)](https://github.com/PennyLaneAI/catalyst/pull/1489)
+
+* Several changes have been made to reduce compile time:
+  * MLIR's verifier has been turned off.
     [(#1513)](https://github.com/PennyLaneAI/catalyst/pull/1513)
-  - Remove unnecessary I/O.
+  * Unnecessary I/O has been removed.
     [(#1514)](https://github.com/PennyLaneAI/catalyst/pull/1514)
     [(#1602)](https://github.com/PennyLaneAI/catalyst/pull/1602)
-  - Sort improvements to reduce complexity and memory.
+  * Improvements have been made to reduce complexity and memory.
     [(#1524)](https://github.com/PennyLaneAI/catalyst/pull/1524)
-  - Lazy IR canonicalization and LLVMIR textual generation.
+  * IR canonicalization and LLVMIR textual generation is now performed lazily.
     [(#1530)](https://github.com/PennyLaneAI/catalyst/pull/1530)
   - Speed up how tracers are overwritten for hybrid ops.
```

Comment:
```suggestion
  * Speed up how tracers are overwritten for hybrid ops.
```

Reply:
Just for consistency with the above changes. Having said that, I actually like using different symbols for the first and second levels, I think that improves readability!